### PR TITLE
fix(3341): handleDownloads function is broken with Puppeteer 14.4 and onwards

### LINKED ISSUE #3341: codeceptjs/puppeteer@14.4.1 fails on handleDownloads()
#### What are you trying to achieve?
Use handleDownloads() to download file in test with codeceptjs/puppeteer

#### What do you get instead?
On I.handleDownloads() there is the following error:
```
     this.page._client.send is not a function
      at Puppeteer.handleDownloads (node_modules/codeceptjs/lib/helper/Puppeteer.js:1072:30)
      at Step.run (node_modules/codeceptjs/lib/step.js:122:47)
      at /content/n/d3/node_modules/codeceptjs/lib/actor.js:134:23

```
### Details

* CodeceptJS version: 3.3.3
* NodeJS Version: 16.15.1
* Operating System: Linux (WSL2)
* puppeteer: 14.4.1
* Configuration file:
```js
{
  tests: ['./test/*.js', './test/testdir/*.js'],
  output: './test/output',
  helpers: {
    Puppeteer: {
      url: 'http://localhost:8080',
      show: true,
      windowSize: '1200x900',
      chrome: {
        args: ['--no-sandbox', '--window-size=1280,960'],
        defaultViewport: {
          width: 1280,
          height: 960
        }
      }
    },
    FileSystem: {},
    clickIfVisible: {
      require: './test/helpers/click-if-visible.js',
    },
  },
  include: {
    I: './steps_file.js'
  },
  bootstrap: null,
  mocha: {},
  name: 'LAP',
  plugins: {
    pauseOnFail: {},
    retryFailedStep: {
      enabled: true
    },
    screenshotOnFail: {
      enabled: true
    }
  }
}
```

My test worked fine and now it fails on 'I.handleDownloads() with the error shown above.
All I changed was the puppeteer version.

Everything is fine with puppeteer@14.3.0 (and previous versions).
Since puppeteer@14.4.0 (including current 14.4.1) the test always fails.

### PR DESCRIPTION
**Pull request by kobenguyent**
## Motivation/Description of the PR
- Resolves #3341 

Applicable helpers:
- [x] Puppeteer

## Type of change
- [x] :bug: Bug fix

## Checklist:
- [x] Lint checking (Run `npm run lint`)
- [x] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -84,13 +84,14 @@
     "ms": "^2.1.3",
     "parse-function": "^5.6.4",
     "promise-retry": "^1.1.1",
+    "read-package-json": "^5.0.1",
```
Comment (DavertMik) [inline package.json]: why do we need this package?

File: lib/helper/Puppeteer.js [hunk 2]
```diff
@@ -1069,6 +1071,12 @@ class Puppeteer extends Helper {
       fs.mkdirSync(downloadPath, '0777');
     }
     fsExtra.emptyDirSync(downloadPath);
+    // Backwards compatibility with Puppeteer < 14.4.0
+    if (parseInt(version.split('.')[0], 10) >= 14) {
```
Comment (DavertMik) [inline lib/helper/Puppeteer.js]: can we just check for >= 15 ?

also I propose a better syntax to handle version:
```js
const [majorVersion] = version.split('.');
if (majorVersion >= 15)  ...
```

**Review by DavertMik — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kobenguyent) [reply package.json]: well, this one to read the version of any package easier

File: lib/helper/Puppeteer.js (hunk 2 above)
Comment (kobenguyent) [reply lib/helper/Puppeteer.js]: I'm just wanting to do the backwards compatibility, like 14.3 and backwards should be not broken. So do you think we only care the major version, don't you?

Comment (DavertMik) [reply lib/helper/Puppeteer.js]: yes, 15.x is current one, we should not bother of some minor like 14.x

File: package.json (hunk 1 above)
Comment (DavertMik) [reply package.json]: no, I'm against adding extra packages without any real need. 
please consider a workaround

**Review by DavertMik — CHANGES_REQUESTED** [review]

**Comment by kobenguyent** [comment]
updated @DavertMik!

**Comment by DavertMik** [comment]
Thanks!

**Pull request merged**